client/src/components/Modal/HeaderModal.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import styled from "styled-components";
+
+const ModalHead = styled.header`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 0.5rem;
+  position: sticky;
+  top: 0;
+  background-color: #fff;
+  padding: 1rem;
+  input {
+    border: none;
+    border-bottom: 1px solid black;
+    margin-right: 1rem;
+    padding: 0.3rem;
+    font-family: "Orbitron", Arial, sans-serif;
+  }
+`;
+
+const HeaderModal = (setModalOpen, modalOpen, onSubmit, value, onChange) => {
+  return (
+    <ModalHead>
+      <button onClick={() => setModalOpen(!modalOpen)}>✖️</button>
```

Comment:
its not a good convention to give the state and the function to change it into the component, its not controlled anymore